setup.py

```diff
@@ -177,6 +177,7 @@ def hotpatch_var(var, prefix='USE_'):
 from tools.setup_helpers.mkldnn import (USE_MKLDNN, MKLDNN_LIBRARY,
                                         MKLDNN_LIB_DIR, MKLDNN_INCLUDE_DIR)
 from tools.setup_helpers.nnpack import USE_NNPACK
+from tools.setup_helpers.qnnpack import USE_QNNPACK
```

Comment:
Please add `'QNNPACK'` to use_env_vars above as well as `NO_QNNPACK` entry to the top description. This will let people set `USE_QNNPACK` or `NO_QNNPACK` and things will still work.

Reply:
done

Reply:
At the very very top (in the comments) do you mind adding 

```
 #
 #   NO_QNNPACK
 #     disables QNNPACK build (quantized 8-bit operators)
```
We try and keep that updated for end users.